modules/mono/editor/script_templates/VisualShaderNodeCustom/basic.cs

```diff
@@ -3,6 +3,8 @@
 using _BINDINGS_NAMESPACE_;
 using System;
 
+[Tool]
+[GlobalClass]
```

Comment:
Should be one line no? Like `[Tool, Global class]`?

CC @raulsntos

Reply:
Both work, I'm not sure that we have a preference.

Since this is used as a template, I don't know what would be less confusing for users. I'm leaning towards leaving the attributes in separate attribute lists, it seems like it may be slightly more common.

Reply:
I was actually leaning towards using both in one statement, but I do believe that it's less confusing to have them as seperate attributes - it's usually how people declare them anyways.